chart/kube-flannel/.helmignore

```diff
@@ -0,0 +1 @@
+tests/
```

Comment:
Could you also add `.helmignore` between the files to ignore when creating the chart.
```suggestion
.helmignore
tests/
```

Reply:
done